sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -479,6 +479,13 @@ public void captureSession(final @NotNull Session session, final @Nullable Hint
       return;
     }
 
+    if (session.isAppInForeground() != null && !session.isAppInForeground()) {
```

Comment:
Maybe replace this with
`if (!Boolean.TRUE.equals(session.isAppInForeground()))`?